pages/[content].tsx

```diff
@@ -67,24 +71,37 @@ const ContentPage: NextPage = (props) => {
 	)
 }
 
-export const getStaticPaths: GetStaticPaths = async () => {
-	const paths = pages
-		.filter((page) => page.content)
-		.map((page) => ({
-			params: { content: page.content },
-		}))
+export const getStaticPaths: GetStaticPaths = async (context) => {
+	const locales = context.locales!
+	const paths = pages.filter((page) => page.content)
+	// .map((page) => ({
```

Comment:
Is this on purpose or some leftover?

Reply:
Leftover. Thanks for noticing! I’ll remove it